ambry-rest/src/test/java/com.github.ambry.rest/NettyResponseChannelTest.java

```diff
@@ -467,6 +467,37 @@ public void keepAliveTest() {
   }
 
   /**
+   * Tests that client initiated terminates don't count towards {@link HttpResponseStatus#INTERNAL_SERVER_ERROR}.
+   */
+  @Test
+  public void clientEarlyTerminateTest() throws Exception {
+    EmbeddedChannel channel = createEmbeddedChannel();
+    TestingUri uri = TestingUri.OnResponseCompleteWithEarlyClientTerminate;
+    HttpRequest httpRequest = RestTestUtils.createRequest(HttpMethod.POST, uri.toString(), null);
+    HttpUtil.setKeepAlive(httpRequest, false);
+
+    String iseMetricName = MetricRegistry.name(NettyResponseChannel.class, "InternalServerErrorCount");
+    long iseBeforeCount = MockNettyMessageProcessor.METRIC_REGISTRY.getCounters().get(iseMetricName).getCount();
+    String citMetricName = MetricRegistry.name(NettyResponseChannel.class, "ClientEarlyTerminateCount");
```

Comment:
`cetMetricName`?

Reply:
Not sure why I put an "i" instead of "e". Fixed, thanks !